README.md

```diff
@@ -99,10 +104,24 @@ sudo npm install -g ijavascript
 ijsinstall
 ```
 
-In Windows, [Anaconda](http://continuum.io/downloads) offers a convenient
-distribution to install Python and many other packages, such as Jupyter and
-IJavascript.
+### Windows
+Install [Python3](https://www.python.org/downloads/) or the [Anaconda3](http://continuum.io/downloads) Python distribution.
+
+In the command line:
```

Comment:
To avoid ambiguity, instead of having a section for `Windows`, let's have two sections: `Window (Official Python distribution)` and `Windows (Anaconda distribution)`.